.github/workflows/build_and_deploy_docu.yml

```diff
@@ -2,10 +2,9 @@ name: Build and deploy docu
 
 on:
   push:
-    branches:
-      - 'main'
```

Comment:
Do you really intend to build and deploy also for branches other than main?

Reply:
I thought about that too, but otherwise, the CI will not be triggered, right?

Reply:
Why not just giving it a try? You'll see what happens to your PR.

Reply:
Anyway, do you want a docu push on *every* PR?

Reply:
I will undo the changes and apply your recommendation in another issue / branch.